packages/tizen_rpc_port/lib/src/port.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:async';
+import 'dart:typed_data';
+
+import 'parcel.dart';
+import 'rpc_port_method_channel.dart';
+
+/// The port type.
+enum PortType {
+  /// The main port type.
+  main,
+
+  /// The callback port type.
+  callback,
+}
+
+/// The port connection between proxy & stub.
+class Port {
+  /// Create port from proxy.
+  Port.fromProxy(this.portName, this.instance, this.portType) : appid = '';
+
+  /// Create port from stub.
+  Port.fromStub(this.appid, this.portName, this.portType) : instance = '';
```

Comment:
Named parameter seems better for usability here.

```suggestion
  /// Create port from proxy.
  Port.fromProxy({
    required this.instance,
    required this.portName,
    required this.portType,
  }) : appid = '';

  /// Create port from stub.
  Port.fromStub({
    required this.appid,
    required this.portName,
    required this.portType,
  }) : instance = '';
```